apps/meteor/app/livechat/server/hooks/saveLastMessageToInquiry.ts

```diff
@@ -0,0 +1,35 @@
+import { isOmnichannelRoom } from '@rocket.chat/core-typings';
+
+import { callbacks } from '../../../../lib/callbacks';
+import { LivechatInquiry } from '../../../models/server/raw';
+import { settings } from '../../../settings/server';
+import { RoutingManager } from '../lib/RoutingManager';
+
+callbacks.add(
+	'afterSaveMessage',
+	async (message, room) => {
```

Comment:
```suggestion
	async (message: IMessage, room: IRoom) => {
```

Reply:
in this case you could also use `isEditedMessage` from `IMessage.ts`

Reply:
It's actually detecting the right typings from the "aftersavemessage" callback signature, so i think we can skip these 👀

Reply:
cool.. I thought it was needed to add them to avoid the `(message as any)` in the lines below